app/pom.xml

```diff
@@ -653,6 +640,197 @@
       </properties>
     </profile>
 
+    <!-- Sonar Cloud and Coverage -->
+    <profile>
+      <id>sonarcloud</id>
+      <properties>
+        <basepom.check.fail-coverage>false</basepom.check.fail-coverage>
+        <basepom.check.skip-coverage>false</basepom.check.skip-coverage>
+      </properties>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-surefire-plugin</artifactId>
+            <configuration>
+              <!-- Fix "There was a timeout or other error in the fork" errors -->
+              <forkedProcessTimeoutInSeconds>120</forkedProcessTimeoutInSeconds>
```

Comment:
120 is defined in our main pom.xml: https://github.com/syndesisio/syndesis/blob/1.12.x/app/pom.xml#L999-L1006
Maybe it is not needed anymore?

I added here because I needed to add the following surefireArgLine.

Reply:
Yeah, probably a mistake we made in not configuring via basepom. For simplicity and consistency we should tweak basepom properties rather than changing the plugin configuration directly.

Reply:
Moved to the properties on the top of pom.xml